R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -2236,12 +2236,15 @@ test_that("gapply() on a DataFrame", {
   actual <- collect(df1)
   expect_identical(actual, expected)
 
+  df1Collect <- gapplyCollect(df, list("a"), function(key, x) { x })
```

Comment:
maybe better change list("a") to "a" to test if a scalar column parameter can work